Source/GSNibLoading.m

```diff
@@ -2005,22 +2004,33 @@ proxy objects (-1 = File's Owner, -2 = First Responder,
     {
       NSInteger oid = [(id)NSMapGet(_oids, obj) intValue];
       if (oid >= 0)
-        {
-          if ([obj respondsToSelector: @selector(realObject)])
-            {
-              obj = [obj realObject];
-            }
-          if ([obj respondsToSelector: @selector(awakeFromNib)])
-            {
-              [obj awakeFromNib];
-            }
-        }
+	{
+	  if ([obj respondsToSelector: @selector(realObject)])
+	    {
+	      obj = [obj realObject];
+	    }
+
+	  // Awaken the object....
+	  [obj awakeFromNib];
+
+	  // If we are in Gorm/IB, set the object up for display
+	  if ([NSClassSwapper isInInterfaceBuilder] == YES)
+	    {
+	      [obj prepareForInterfaceBuilder];
+	    }
+	}
     }
 
   // awaken the owner
   if ([owner respondsToSelector: @selector(awakeFromNib)])
     {
       [owner awakeFromNib];
+
+      // If we are in Gorm/IB, set the object up for display
+      if ([NSClassSwapper isInInterfaceBuilder] == YES)
+	{
+	  [obj prepareForInterfaceBuilder];
```

Comment:
I think that `obj` will be undefined here. Did you mean `owner`? Not sure whether that is needed.

Reply:
I changed this to owner.